data/preset/gallery.yaml

```diff
@@ -0,0 +1,555 @@
+                                      other: CG2
+                                    slots:
+                                      sub-filter:
+                                        type: Filter.OtherGroup
+                                        references:
+                                          other: CG3
+                                        slots:
+                                          sub-filter:
+                                            type: Filter.OtherGroup
+                                            references:
+                                              other: CG4
+                                            slots:
+                                              sub-filter:
+                                                type: Filter.OtherGroup
+                                                references:
+                                                  other: CG5
+                  - type: ContentGroup.ContentGroup
+                    slots:
+                      arrangement:
+                        <<: *main-arrangement
+                      selection:
+                        <<: *selection-set
+                        id: CG7
+                        slots:
+                          filter:
+                            type: Filter.Articles
+                            slots:
+                              sub-filter:
+                                type: Filter.OtherGroup
+                                references:
+                                  other: CG1
+                                slots:
+                                  sub-filter:
+                                    type: Filter.OtherGroup
+                                    references:
+                                      other: CG2
+                                    slots:
+                                      sub-filter:
+                                        type: Filter.OtherGroup
+                                        references:
+                                          other: CG3
+                                        slots:
+                                          sub-filter:
+                                            type: Filter.OtherGroup
+                                            references:
+                                              other: CG4
+                                            slots:
+                                              sub-filter:
+                                                type: Filter.OtherGroup
+                                                references:
+                                                  other: CG5
+                                                slots:
+                                                  sub-filter:
+                                                    type: Filter.OtherGroup
+                                                    references:
+                                                      other: CG6
```

Comment:
note for the future: we really need the infinite scrolling version of Arrangement.Constrained